### PR TITLE
[TOOLS-617] Some users avatars doesn't show up in taskboard cards

### PR DESCRIPTION
**Pull request by jytakano**

### CODE REVIEW

File: src/main/java/objective/taskboard/jira/endpoint/JiraEndpoint.java [hunk 1]
```diff
@@ -152,6 +155,28 @@ public Response intercept(Chain chain) throws IOException {
...
+            inputStream.read(bytes);
+            return bytes;
+        } catch (Exception e) {
+            log.error("Error reading bytes from url: " + e.getMessage());
+            throw new IllegalStateException(e);
+        } finally {
+            try {
+                if (inputStream != null)
+                    inputStream.close();
+            } catch (IOException e) {
```
Comment (objective-solutions-user) [inline src/main/java/objective/taskboard/jira/endpoint/JiraEndpoint.java]: MAJOR Either log or rethrow this exception. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3AS1166)

**Comment by objective-solutions-user** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/main/java/objective/taskboard/jira/endpoint/JiraEndpoint.java [hunk 2]
```diff
@@ -152,6 +156,28 @@ public Response intercept(Chain chain) throws IOException {
...
+                throw new UncheckedIOException(e);
```
Comment (objective-solutions-user) [inline src/main/java/objective/taskboard/jira/endpoint/JiraEndpoint.java]: CRITICAL Remove this throw statement from this finally block. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3AS1143)
MAJOR Refactor this code to not throw exceptions in finally blocks. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3AS1163)

**Comment by objective-solutions-user** [comment]
SonarQube analysis reported 2 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/main/java/objective/taskboard/jira/endpoint/JiraEndpoint.java [hunk 3]
```diff
@@ -152,6 +155,24 @@ public Response intercept(Chain chain) throws IOException {
         }
     }
 
+    public byte[] readBytesFromURL(URL url, String username, String password) throws IOException {
+        InputStream inputStream = null;
+        try {
+            URLConnection connection = url.openConnection();
+            connection.setRequestProperty("Authorization", Credentials.basic(username, password));
+            inputStream = connection.getInputStream();
```
Comment (herbert-venancio) [inline src/main/java/objective/taskboard/jira/endpoint/JiraEndpoint.java]: fica menos código usando o try-with-request:
        URLConnection connection = url.openConnection();
        connection.setRequestProperty("Authorization", Credentials.basic(username, password));
        try (InputStream inputStream = connection.getInputStream()) {
...
        } catch (Exception e) {
...
        }

**Pull request merged**